### ISSUE TITLE
Animated avatars play constantly without option to stop them (WCAG 2.1 / 2.2.2 violation)

### ISSUE THREAD

**Issue by Mikaela** [vector-im/element-web#18442]
### Steps to reproduce

1. use a homeserver with animations enabled e.g. the-apothecary.club
2. join a room with someone who has animated avatar (e.g. #matrix:matrix.org)
3. see someone with an animated avatar say something

### What happened?

The animation plays constantly and there is no way to stop it, so focus moves from the messages to it.

### What did you expect?

Element either doesn't play it constantly or has an option to stop it. See suggestions at https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html

### Operating system

Fedora 34

### Browser information

Microsoft Edge 93.0.961.11 (Virallinen koontiversio) beta (64-bittinen)

### URL for webapp

develop.element.io

**Comment by Mikaela** [comment]
Originally reported at https://github.com/vector-im/element-web/issues/15790#issuecomment-894664308 where a new issue was requested https://github.com/vector-im/element-web/issues/15790#issuecomment-894664858.

See also:

* https://github.com/vector-im/element-web/issues/6534
* https://github.com/vector-im/element-web/issues/7473
* https://github.com/vector-im/element-web/issues/15790

**Comment by Mikaela** [comment]
I think this is missing the A11y label

**Comment by kittykat** [comment]
Hi @Mikaela do you mean that animations take up user focus or something else like keyboard/screenreader focus?

**Comment by Mikaela** [comment]
User focus, please see e.g. https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html

**Comment by Palid** [comment]
I think we should definitely not play those animations if user has a reduced animations setting in the browser (or/and maybe some app settings too?) but I can't really figure out any good UX to start/stop those without breaking current user experience. Clicking on the avatar already does something, so that'd be a huge UX breaking change.

**Comment by janogarcia** [comment]
@Palid Maybe we could just disable autoplay avatar GIFs if the user has `prefers-reduced-motion` enabled, and then optionally play the animation only on hover for those users? The hover action would work as the start/stop control.

**Comment by HarHarLinks** [comment]
this includes the use of avatars as blurred background to the room list.
